src/main/resources/db/migration/data/R__3_2__AGENCY_INTERNAL_LOCATIONS.sql

```diff
@@ -315,3 +316,13 @@ VALUES
 (-206, '1',    'LAND',  'SYI',   'NA', 'Y', 20, 20, 'SYI-H-1', -13, 'Landing H/1'),
 (-207, '1',    'CELL',  'SYI',   'NA', 'Y',  1,  1, 'SYI-H-1-1', -14, 'Cell H/1-1'),
 (-208, 'CHAP', 'AREA',  'SYI',   'NA', 'N',  1,  1, 'SYI-CHAP', null, 'Chapel');
+
+-- 'Abstract' locations in LEI for 'Prisoner's cell' and 'Other cell'.  selected by event type 'OCCUR', used by the Use-of-Force application. See DCS-424
+-- I production these locations will be repeated for every agency that uses 'use-of-force'
+
+INSERT INTO AGENCY_INTERNAL_LOCATIONS (INTERNAL_LOCATION_ID, INTERNAL_LOCATION_CODE, INTERNAL_LOCATION_TYPE, AGY_LOC_ID, UNIT_TYPE, CERTIFIED_FLAG, CAPACITY, OPERATION_CAPACITY, DESCRIPTION, PARENT_INTERNAL_LOCATION_ID, USER_DESC)
+VALUES
+(1900, 'PRISONERSCEL', 'CELL', 'LEI', 'NA', 'Y', 10, 10, 'LEI-OTHER-PRISONERSCEL', 13410, 'Prisoners cell'),
```

Comment:
Does that need to say 'Prisoner's cell' with an apostrophe?

Reply:
Yes, it does.